### PR TITLE
DevOps pipeline update for RSU Manager

### PR DESCRIPTION
**Pull request by drewjj**
Includes pipeline support for:
- New cloud functions
- RSU Manager new environment variables

### CODE REVIEW

File: cdot-web-app/src/EnvironmentVars.js [hunk 1]
```diff
@@ -9,14 +9,24 @@ class EnvironmentVars {
...
+
     static rsuInfo = `${this.getBaseApiUrl()}/rsuinfo`;
     static rsuOnline = `${this.getBaseApiUrl()}/rsu-online-status`;
     static rsuCounts = `${this.getBaseApiUrl()}/rsucounts`;
     static googleAuth = `${this.getBaseApiUrl()}/rsu-google-auth`;
     static rsuRestAuth = `${this.getBaseApiUrl()}/rsu-rest-auth`;
 
     static MAPBOX_TOKEN = process.env.REACT_APP_MAPBOX_TOKEN;
-    static GOOGLE_CLIENT_ID = process.env.REACT_APP_GOOGLE_CLIENT_ID;
+    static GOOGLE_CLIENT_ID = this.getClientId();
```
Comment (payneBrandon) [inline cdot-web-app/src/EnvironmentVars.js:29]: Was this not switching based on environment already?

**Review by payneBrandon — APPROVED** [review]
Looks good to me, just the one comment!

File: cdot-web-app/src/EnvironmentVars.js (hunk 1 above)
Comment (drewjj) [reply cdot-web-app/src/EnvironmentVars.js:29]: No, not really. The idea was that each environment would use the same environment variable name but with a different value. What I am noticing in the RSU Manager pipeline is that there pretty much needs to be separate variables for each environment similar to the RSU Gateway ().

Comment (payneBrandon) [reply cdot-web-app/src/EnvironmentVars.js:29]: I wonder if we're just not properly building for each environment then. I wouldn't think we'd have to have separate variable names for each environment like that. We'll want to check our ci/cd pipelines and make sure we're specifying the environment when building each.

Comment (payneBrandon) [reply cdot-web-app/src/EnvironmentVars.js:29]: Are you using the build and release pipelines to deploy or doing this manually? The build pipeline looks like it should be creating appropriate environment-specific builds running the `env-cmd -f <filename>...` syntax - if that's not working we definitely need to dig into why and how to get those builds working properly.

**Review by payneBrandon — APPROVED** [review]
Looks good to me!

**Pull request merged**